tensorflow/python/kernel_tests/random/random_binomial_test.py

```diff
@@ -207,6 +208,24 @@ def moment(self, i):
         )
         self.assertAllLess(z_scores, z_limit)
 
+  def testStatelessDtypes(self):
+    # Test case for GitHub issue 59160
```

Comment:
nit: remove this comment
Ideally the test name should explain itself, therefore there is no need to include github issue id here.

Reply:
Thanks @vufg. The comment has been removed.